composer.json

```diff
@@ -23,15 +23,15 @@
   ],
   "require": {
     "php": ">=7.1",
-    "fzaninotto/faker": "^1.4",
-    "intervention/image": "^2.4"
+    "fakerphp/faker": "^1.13.0",
+    "intervention/image": "^2.5.1"
   },
   "autoload": {
     "psr-4": {
       "Salopot\\ImageGenerator\\": "src"
     }
   },
   "require-dev": {
-    "phpunit/phpunit": "^7.5||^8.0"
+    "phpunit/phpunit": "^9.5.2"
```

Comment:
I think we should left at list one old version for back compatibility
`^8.0||^9.5`